rules/windows/builtin/security/win_security_susp_possible_shadow_credentials_added.yml

```diff
@@ -0,0 +1,25 @@
+title: Possible Shadow Credentials Added
+id: f598ea0c-c25a-4f72-a219-50c44411c791
+status: experimental
+description: Detects possible addition of shadow credentials to an active directory object.
+author: Nasreddine Bencherchali (rule), Elastic (idea)
+references:
+    - https://www.elastic.co/guide/en/security/7.17/prebuilt-rule-0-16-1-potential-shadow-credentials-added-to-ad-object.html
+    - https://cyberstoph.org/posts/2022/03/detecting-shadow-credentials/
+    - https://twitter.com/SBousseaden/status/1581300963650187264?
+date: 2022/10/17
+logsource:
+    product: windows
+    service: security
+    definition: The "Audit Directory Service Changes" logging policy must be configured in order to recieve events. Audit events are generated only for objects with configured system access control lists (SACLs), and only when they are accessed in a manner that matches their SACL settings. This policy covers the following events ids - 5136, 5137, 5138, 5139, 5141
+detection:
+    selection:
+        EventID: 5136
+        AttributeLDAPDisplayName: 'msDS-KeyCredentialLink'
```

Comment:
Samir suggested a tuning on the logic which is pending review, but it seems to make it more reliable against FPs
https://github.com/elastic/detection-rules/pull/2359

Reply:
Interesting change. While it reduces the FP rate it also downgrades the rule in a lot of ways as with the simple change you could bypass it.

I don't have good visibility on the FP from this kind of rule so I don't know if it's worth hardening the rule to only Offensive OSS tooling.

Reply:
I've added a commented out section explaining the filters that could be added if FPs are encountered. I think it's a better way to handle this. (At least for now)